sdks/python/apache_beam/internal/pickler.py

```diff
@@ -136,6 +136,32 @@ def _reject_generators(unused_pickler, unused_obj):
 
 dill.dill.Pickler.dispatch[types.GeneratorType] = _reject_generators
 
+if sys.version_info[0] > 2:
+  # Monkey patch for dill._dill.Pickler to pickle functions
```

Comment:
Could we add a TODO to remove this after https://github.com/uqfoundation/dill/issues/313 is fixed?

Reply:
Added.